packages/libs/eda/src/lib/core/hooks/analysis.ts

```diff
@@ -448,6 +503,15 @@ export function useAnalysisList(analysisClient: AnalysisClient) {
     );
   }, [analysisClient]);
 
+  useEffect(() => {
+    loadList();
+  }, [loadList]);
+
+  useEffect(() => {
+    // TODO Merge in change!
```

Comment:
Also, could these two useEffects with the same dependency arrays be combined into one?

Reply:
The `TODO` is to update the list in state, rather than refetch the list from the backend. It's an optimization, which I may do before merging this PR. Thank you for the reminder!